app/code/Magento/Sales/Block/Adminhtml/Order/View/Tab/Invoices.php

```diff
@@ -35,7 +47,10 @@ public function getTabTitle()
      */
     public function canShowTab()
     {
-        return true;
+        if($this->authorization->isAllowed('Magento_Sales::invoice'))
```

Comment:
it is fine to shorten this construction to `return $this->authorization->isAllowed('Magento_Sales::creditmemo');`